### PR TITLE
Update release script not to upload insecurly

### PR DESCRIPTION
**Pull request by Carreau**
All in title.

### CODE REVIEW

File: tools/release [hunk 1]
```diff
@@ -51,17 +51,19 @@ if 'upload' not in sys.argv:
...
+# sh('./setup.py register')
 
 # Upload all files
-sh(sdists + ' upload')
+sh(sdists)
 for py in ('2.7', '3.4'):
-    sh('python%s setupegg.py bdist_wheel upload' % py)
+    sh('python%s setupegg.py bdist_wheel' % py)
+
+print('Will not upload with setuptools as upload connection is insecure. Please use `twine upload dist/*` to upload the files to PyPI')
```
Comment (takluyver) [inline tools/release]: Why not call twine in the script?

File: tools/release [hunk 2]
```diff
@@ -51,17 +51,19 @@ if 'upload' not in sys.argv:
     sys.exit(0)
 
 # Register with the Python Package Index (PyPI)
-print( 'Registering with PyPI...')
+print( 'Not Registering with PyPI, registering with setup.py is insecure as communication is not encrypted')
```
Comment (takluyver) [inline tools/release]: I don't think we need to print a message saying that we're not doing this. A comment is fine. And the call can be removed rather than just commented.

File: tools/release (hunk 1 above)
Comment (Carreau) [inline tools/release]: Sure in the long run, but we have a people visiting today in 1/2 an hour, and I was going to make that in a separate PR when I have time unless So does it in the meantime. (I guess you can use twine as a library). 

Also I'm sure there will be much bikeshed so separate seem better to iterate quickly.

File: tools/release (hunk 2 above)
Comment (Carreau) [inline tools/release]: Done.

File: tools/release (hunk 1 above)
Comment (takluyver) [inline tools/release]: I was thinking of just putting `sh('twine upload dist/*')` in there. No need to work out how to use it as a library. ;-)
